testing/trino-testing/src/main/java/io/trino/testing/DistributedQueryRunner.java

```diff
@@ -774,6 +800,11 @@ protected SELF self()
         public DistributedQueryRunner build()
                 throws Exception
         {
+            String tracingEnabled = firstNonNull(getenv("TRACING"), "false");
```

Comment:
let's maybe use a more unique name for this variable to avoid potential collisions.

Reply:
https://github.com/trinodb/trino/pull/18307